core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -473,6 +476,11 @@ private[spark] object CoarseGrainedExecutorBackend extends Logging {
       }
 
       driverConf.set(EXECUTOR_ID, arguments.executorId)
+
+      if (driverConf.get(EXECUTOR_ALLOW_SYNC_LOG_LEVEL)) {
+        // Set the current log level of driver in Executor
+        cfg.logLevel.foreach(l => updateLogLevel(l))
+      }
```

Comment:
```suggestion
      cfg.logLevel.foreach(l => updateLogLevel(l))
```
I think the check is not needed as when `EXECUTOR_ALLOW_SYNC_LOG_LEVEL` is false then the `cfg.logLevel` will be `None`.